console/console-init/ui/src/modules/address-space/components/EndpointCertificateConfiguration/EndpointCertificateConfiguration.tsx

```diff
@@ -25,7 +25,7 @@ const EndpointCertificateConfiguration: React.FunctionComponent<IEndpointCertifi
     <Form>
       <FormGroup
         label="Certificate"
-        fieldId="text-file-uplaod-certificate"
+        fieldId="text-file-upload-certificate"
```

Comment:
PF guidelines suggest that fielId and the corresponding input element should be same. Is there a reason for deviating here?